README.md

```diff
@@ -50,15 +50,17 @@ An example cross platform middle tier api is provided as part of the solution, i
     - [Installing the Linux Subsystem on Windows](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
 3. Docker
     - [Install on Mac](https://docs.docker.com/docker-for-mac/install/)
-    - [Install on Windows](https://docs.docker.com/docker-for-windows/install/)
+    - [Install on Windows](https://store.docker.com/editions/community/docker-ce-desktop-windows)
+        - Select the "Use Windows containers ..." option
+        - After rebooting, you will be prompted to login to Docker.  You can create a personal username and pasword here.        
```

Comment:
Logging in should be optional, at least on Mac it is.

Reply:
@joelong01 Maybe add something like "After rebooting, you will be prompted to optionally login to Docker"